src/main/java/org/mozilla/jss/CryptoManager.c

```diff
@@ -517,6 +533,251 @@ Java_org_mozilla_jss_CryptoManager_initializeAllNative2
     return;
 }
 
+/* Create a complete version of the initialization method that
+ * launches NSS as a context instead of a full NSS_Init.
+ * this allows the app to assure they don't inherit all the properties
+ * of a previous full init done by the main process.
+ * Allow the caller of JSS to EITHER do a full nit OR do a context init.
+ */
```

Comment:
Might be a good idea to state something like the following:
-----------
Note: This method is modified from a copy of an existing method: "Java_org_mozilla_jss_CryptoManager_initializeAllNative2";
The changes involved are the following:
1. It calls NSS_InitContext(...) instead of NSS_Initialize(...)
2.  The original method has some code to detect readOnly and calls special read only nss init functions, which don't make sense with a context. That logic was therefore removed.
The reason why copy/modify is done instead of making changes to the existing code was because XYZ.
We need to be aware in case there's any bug fixes in the original code that will need to be applied to this method as well

Reply:
Taken care of in most recent push.